lib/api/model/initial_snapshot.dart

```diff
@@ -207,6 +210,22 @@ class InitialSnapshot {
   Map<String, dynamic> toJson() => _$InitialSnapshotToJson(this);
 }
 
+/// A value of [InitialSnapshot.realmTopicsPolicy].
+///
+/// For docs, search for "realm_topics_policy"
+/// in <https://zulip.com/api/register-queue#response>.
+@JsonEnum(fieldRename: FieldRename.snake)
+enum RealmTopicsPolicy {
+  allowEmptyTopic,
+  disableEmptyTopic;
+
+  static RealmTopicsPolicy fromApiValue(String apiValue)=> _byApiValue[apiValue] ?? allowEmptyTopic;
```

Comment:
Instead of handling unknown values here, let's make an explicit `unknown` enum value, and ask the inner parts of the app code to handle that.